library/src/main/kotlin/org/codice/compliance/Common.kt

```diff
@@ -32,7 +32,8 @@ import javax.xml.transform.TransformerFactory
 import kotlin.test.currentStackTrace
 
 const val IMPLEMENTATION_PATH = "implementation.path"
-const val DEFAULT_IMPLEMENTATION_PATH = "implementations/ddf"
+@Suppress("TopLevelPropertyNaming")
+val DEFAULT_IMPLEMENTATION_PATH = "implementations${File.separator}ddf"
```

Comment:
❓ Does this enable you to run the script from the command-line on Windows? That's something we should probably fix pretty soon.

Reply:
It fixes something windows related. I think it was even in the code things were messing up (some tests were failing because they couldn't find a file because of that separator).